content/src/roadmaps/tasks/raffle-bingo-games-license.md

```diff
@@ -0,0 +1,10 @@
+---
+displayname: raffle-bingo-games-license
+urlSlug: raffle-bingo-games-license
+filename: raffle-bingo-games-license
+name: Apply for a Raffle or Bingo Game License
+id: raffle-bingo-games-license
+summaryDescriptionMd: ""
+---
+
+This task is intentionally blank. The content here is controlled by raffle-step-license-1 and raffle-step-license-2.
```

Comment:
[boulder] We should talk to content about this task. Should this actually live with the rest of the license tasks? Right now this will not be sync'd to Webflow, and are they anticipating that it will be? The behavior of the task in-app will not change. This is more of a question of how they want this to work with Webflow.

If that's the case, and they do want it to sync, the content being sync'd to Webflow will be:
```
This task is intentionally blank. The content here is controlled by raffle-step-license-1 and raffle-step-license-2.
```
If they have specific content that they'd like to be displayed on Webflow that should be added here. It will not show up in the Navigator. This is exclusively in regards to data being sync'd.

Reply:
@AnJuHyppolite I know we discussed this with content in Slack and at TLC daily scrum, but did we ever come to a decision here?

@ReginaL1 not mission critical for this piece of work, but just want to make sure it's on your radar.

Reply:
@jphechter @AnJuHyppolite  I believe we punted to decide later what the content for Webflow will be, since it's a complex case. For now, showing the task screens on My Account is the higher priority